src/Utilities(JSON_Data)/Raw_JSON_Data.php

```diff
@@ -0,0 +1,42 @@
+<?php
+
+require('../../vendor/autoload.php');
+use FurnitureStoreAPI\DatabaseConnection\DBConnect as Connection;
+
+$db = Connection::db();
+$categories = [];
+$filename = 'furniture.json';
+$rawData = json_decode(file_get_contents($filename), true);
+// Create a list of unique categories
+foreach ($rawData as $data) {
+    $categories[] = $data['name'];
+}
+$uniqueCategories = array_unique($categories);
+$sortedArray = sort($uniqueCategories);
```

Comment:
Since you've put all this into your `$map` anyway, these lines aren't needed any more